src/configuration/app-containers.md

```diff
@@ -9,19 +9,19 @@ You control your application and the way it will be built and deployed on
 Platform.sh via a single configuration file, `.platform.app.yaml`, located at
 the root of your application folder inside your Git repository.
 
-Here's an example of a minimalist `.platform.app.yaml` file for PHP:
+An example of a minimalist `.platform.app.yaml` file for PHP is below:
```

Comment:
`Here's` doesn't clearly indicate "where" the reader should look.  Below is more understandable.